src/test/java/org/fcrepo/migration/ExportedArchiveFoxmlMigratorTest.java

```diff
@@ -19,9 +23,12 @@
     @Before
     public synchronized void processFoxml() throws XMLStreamException, IOException {
         if (getResult() == null) {
-            result = new DummyHandler();
-            fetcher = new DummyURLFetcher();
-            new Migrator(new SimpleObjectSource("example1-foxml.xml", getFetcher(), new DummyIDResolver()), getResult()).run();
+            final ApplicationContext context = new ClassPathXmlApplicationContext("spring/exported-foxml.xml");
+            this.result = (DummyHandler) context.getBean("dummyHandler");
+            this.fetcher = (DummyURLFetcher) context.getBean("dummyFetcher");
```

Comment:
Why are you not using the Spring test framework here and just injecting all this stuff via annotation?